index.php

```diff
@@ -36,11 +38,14 @@
 $searchResponse = json_decode($searchResponse, true);
 $geocodeResponse = $geocodeResponse->body;
 $geocodeResponse = json_decode($geocodeResponse, true);
+$userResponse = $userResponse->body;
+$userResponse = json_decode($userResponse, true);
 
 echo "<b>Given Base URL  - </b>" . $baseURL . "<br>";
 echo "<b>Hello JSON Test - </b>" . $bodyResponse->status . "<br>";
 echo "<b>Peers JSON Test - </b>" . $peersResponse->count . " peers <br>";
 echo "<b>Status JSON Test - </b>" . $statusResponse['index']['messages']['size'] . " messages <br>";
 echo "<b>Search JSON Test - </b>" . $searchResponse['search_metadata']['hits'] . " hits <br>";
 echo "<b>Geocode JSON Test - </b>" . $geocodeResponse['locations']['Hyderabad']['country_code'] . " country code <br>";
+echo "<b>User JSON Test - </b>" . $userResponse['user']['name'] . " is the Twitter user <br>";
```

Comment:
Can you rename this in the format as 
`<username> has <followers> followers and is following <following> users`

That way we are consuming more content from the API to display here.